parlai/core/worlds.py

```diff
@@ -356,8 +356,9 @@ def parley(self):
     def epoch_done(self):
         done = False
         for a in self.agents:
-            if a.epoch_done():
-                done = True
+            if hasattr(a, 'epoch_done'):
```

Comment:
we could also put this function in the base agent class, returning false so it exists?
https://github.com/facebookresearch/ParlAI/blob/master/parlai/core/agents.py#L8